pkg/service/api/discovery_service.go

```diff
@@ -269,3 +272,15 @@ func loadAPIFromMethods(fullPath string, methods []config.Method, headers map[st
 	}
 	return nil
 }
+
+func (ads *LocalMemoryAPIDiscoveryService) PluginPathChange(filePath string) {
```

Comment:
In Go, the receiver name should be as shorter as possible. Just one character should be enough.